### DISCUSSION TITLE
Stuttering audio

### DISCUSSION THREAD

**Discussion by hydrazoan** [apohl79/audiogridder#1225] [Help]
Hi
I'm trying to use a MacBook Pro (MBP) as a Server, which has it itself UAD2 FireWire box plugged in for running Universal Audio plugins. 

Using WiFi, I'm hoping that I can use them from a 2021 MBP. 

Currently, they are talking to each other, but with the occasional stutter / dropout.

Some possible thoughts on what could be contributing to this:
1) the 2010 MBP currently had no battery, as I wasn't even sure it would make it this far. So yes, it's a bit underpowered.
2) the 2010 MBP is an old machine, relatively speaking (but has outlived its replacement).
3) the 2010 MBP is going through its built in sound card (presumably at 48k), while the 2021 MBP uses a Focusrite usb box, at 44.1k. I don't know about the Bit rates ( but I dare say I could find out ).

Do the sample or bit rates matter in this scenario? Does AudioGridder do its own sample rate conversion? Is there some clever jiggery-pokery going on that syncs the two Macs?

Thanks

**Comment by apohl79** [comment]
The server does not require a sound card and does not use it. The reason for the audio dropouts is very likely the network performance. Latency on wifi is too high most of the time, so you better try a wired connection. Larger buffers in AG might get you a bit further.

**Comment by pbl4845** [comment]
I´m experiencing similar situation over cable. What is in common is I am using:
-MBP(2017) with the plugin
-Imac(2013) with Focusrite Liquid Mix Firewire connected VST running the server
-both machines connected with an RJ45 cable
In my case it starts sounding fine, but as time pass by or more instances are added, audio starts to sound stuttered. Looks like a buffer is being filled or something. I am in 1.2 version and have tried every block/buffer combination possible. 
I also have noticed Apple driver kit process with high CPU usage increases (huge CPU compsumption more than Ableton live). 
How can I troubleshoot this further? where can I find logs?
